src/train.py

```diff
@@ -61,12 +62,19 @@
         mask_tiled_uri=io_parameters.mask_tiled_uri,
         mask_tiled_api_key=io_parameters.mask_tiled_api_key,
         is_training=True,
+        using_qlty=False,
```

Comment:
Are we introducing this variable to retain the possibility to go back to the old version of the TiledDataSet, it seems to always be `using_qlty=False`?

Reply:
Yes exactly. This optional parameter is hardcoded to turn off all qlty related operation in `TiledDataset`, while not losing its old functionality, in case we change our mind in the future.